protocols/v2/binary-sv2/serde-sv2/src/primitives/sequences/seq0255.rs

```diff
@@ -230,7 +243,15 @@ impl<'s> Seq0255<'s, U256<'s>> {
                 data: Some(data),
             }
         } else {
-            panic!()
+            // this is an already valid seq should be safe to call the unwraps.
+            // also this library shouldn't be used for priduction envs so is ok do thigs like this
+            // one
+            let data = self.seq.unwrap().parse().unwrap();
```

Comment:
why is this unwrap safe?

Reply:
Same as above, if `self.data` is `None` then `self.seq` is `Some()`

Reply:
Hm, that sounds risky. Here we also have two unwraps. 
maybe something like this:
```rust
impl<'s> Seq0255<'s, u32> {
    pub fn into_static(self) -> Seq0255<'static, u32> {
        match (self.data, self.seq) {
            (Some(data),_) => {
            },
            (_, Some(seq)) => {
            },
            _ => {
                debug_assert!(false, "This is unreachable because `req` most be defined if `data` is `None`");
                unreachable!();
            }

        }
    }
}
```

Reply:
I believe because this library is used only for the MessageGenerator and the seq variant is never used in that case, but we should ask @fi3 for that

Reply:
we need data "variant" for it to be static we can't have a static seq